shared/qt/idian/include/Idian/Row.hpp

```diff
@@ -40,6 +40,9 @@ class GenericRow : public QFrame, public Utils {
 
 public:
 	GenericRow(QWidget *parent = nullptr) : QFrame(parent), Utils(this) { setAccessibleName(""); };
+
+	virtual void setTitle(const QString &title) = 0;
```

Comment:
`title` and not `name` as in the override?

Reply:
Should probably be either `title` or `name` everywhere (currently it's a bit mixed). Which would you prefer?

Reply:
I decided to go with `title` everywhere (it already was `title` in most places and I think it fits better).